idf_component_examples/esp_matter_light/main/idf_component.yml

```diff
@@ -0,0 +1,13 @@
+dependencies:
+  espressif/esp_matter:
+    version: "^1.3.0"
+  # Adds Arduino Core from GitHub repository using main branch
+  espressif/arduino-esp32:
+    # version: "^3.0.5"
+    # override_path: "../../../"
+    # pre_release: true
```

Comment:
@SuGlider you can remove the comment before merging and releasing 3.0.5

Reply:
Done. It is ready.